[Bug] PDF parsing fails on Linux ARM64 Docker due to missing @napi-rs/canvas native binding (DOMMatrix is not defined)

### 📱 Client Type

Web (Desktop Browser)

### 💻 Operating System

macOS

### 📦 Deployment Platform

Self hosting Docker

### 📌 Version

2.1.30

### 🌐 Browser

Safari

### 🐛 Bug Description

### Bug description

When deploying LobeHub/LobeChat using Docker on a Linux ARM64 environment, uploading and parsing a PDF file fails. The server throws a `ReferenceError: DOMMatrix is not defined` error. 

Based on the server logs, the root cause appears to be the missing native binding for `@napi-rs/canvas` on the `linux-arm64-gnu` architecture (specifically, it cannot find `./skia.linux-arm64-gnu.node`).

### Steps to reproduce

1. Deploy LobeChat (v2.1.3) via `docker-compose` on a Linux ARM64 machine.
2. Open the chat interface.
3. Upload a `.pdf` file to the chat.
4. The frontend shows a parsing error, and the backend logs show the missing module error and `DOMMatrix` failure.

### Expected behavior

The PDF file should be parsed successfully without native binding errors, just like it does on AMD64/x86_64 architectures.

### Environment

- **Version**: v2.1.3 (Docker)
- **OS**: Linux ARM64 (e.g., Oracle Cloud ARM, Raspberry Pi, etc.)
- **Deployment**: Docker Compose
- **Database**: PostgreSQL (pgvector) + RustFS (S3)

### Server Logs

```log
Error importing @napi-rs/canvas: Error: Cannot find native binding. npm has a bug related to optional dependencies ([https://github.com/npm/cli/issues/4828](https://github.com/npm/cli/issues/4828)). Please try `npm i` again after removing both package-lock.json and node_modules directory.
    at ignore-listed frames {
  [cause]: Error: Cannot find module '@napi-rs/canvas-linux-arm64-gnu'
  ...
    [cause]: Error: Cannot find module './skia.linux-arm64-gnu.node'
  ...
Warning: Cannot load "@napi-rs/canvas" package
Warning: Cannot polyfill `DOMMatrix`, rendering may be broken.
Warning: Cannot polyfill `ImageData`, rendering may be broken.
Warning: Cannot polyfill `Path2D`, rendering may be broken.
[20260218-125134-g1obv7-2.pdf] File parsing failed: ReferenceError: DOMMatrix is not defined
Error [TRPCError]: DOMMatrix is not defined
  code: 'INTERNAL_SERVER_ERROR',
  [cause]: ReferenceError: DOMMatrix is not defined
Error in tRPC handler (lambda) on path: document.parseFileContent, type: mutation

### 📷 Recurrence Steps

<img width="950" height="302" alt="Image" src="https://github.com/user-attachments/assets/4776700d-96f4-4c63-a771-82b2277dccc0" />

### 🚦 Expected Behavior

_No response_

### 📝 Additional Information

_No response_

### 🛠️ Willing to Submit a PR?

None

### ✅ Validations

- [x] Read the [docs](https://lobehub.com/zh/docs).
- [x] Check that there isn't [already an issue](https://github.com/lobehub/lobe-chat/issues) that reports the same bug to avoid creating a duplicate.
- [x] Make sure this is a LobeChat issue and not a third-party library or provider issue.
- [x] Check that this is a concrete bug. For Q&A, please use [GitHub Discussions](https://github.com/lobehub/lobe-chat/discussions) or join our [Discord Server](https://discord.gg/rGHwKq4R).

--- a/.npmrc
+++ b/.npmrc
@@ -18,3 +18,4 @@ public-hoist-pattern[]=*stylelint*
 
 public-hoist-pattern[]=@auth/core
 public-hoist-pattern[]=pdfjs-dist
+public-hoist-pattern[]=@napi-rs/canvas-*

